src/js/view/controls/nextuptooltip.js

```diff
@@ -71,13 +75,26 @@ define([
             this._api.next();
         }
 
-        toggle(show) {
+        toggle(show, reason) {
             if (!this.enabled) {
                 return;
             }
             dom.toggleClass(this.container, 'jw-nextup-sticky', !!this.nextUpSticky);
-            dom.toggleClass(this.container, 'jw-nextup-container-visible', show);
-            dom.toggleClass(this._playerElement, 'jw-flag-nextup', show);
+            if (this.shown !== show) {
```

Comment:
Shouldn't we only enter this conditional if we have nextUp too? `if (this.shown !== show && this.nextUp)`